src/content/developers/docs/consensus-mechanisms/index.md

```diff
@@ -48,51 +46,63 @@ A consensus mechanism in a cryptoeconomic system also helps prevent certain kind
+Nakamoto consensus is done by [miners](/developers/docs/consensus-mechanisms/pow/mining/) who compete to create new blocks full of processed transactions. They do this through a process known as "proof-of-work" mining (PoW). In PoW, miners are creating random numbers, called a nonce, and plugging that into a hash function to see if they produce a hash which satisfies a condition difficult enough to show they've put work into solving the problem. Once discovered, the miner who found the nonce proposes a new block to the rest of the network. In this new block, the miner pays itself in ETH. If the network decides to continue to build on his block, it becomes part of the "cannonical chain" and the miner has some freshly minted ETH. 
+
+The race is won by whosever computer can solve a math puzzle fastest – this produces the cryptographic link between the current block and the block that went before. Solving this puzzle puts the work in "proof-of-work".
 
 #### Security {#pow-security}
 
 The network is kept secure by the fact that you'd need 51% of the network's computing power to defraud the chain. This would require such huge investments in equipment and energy; you're likely to spend more than you'd gain.
 
 More on [proof-of-work](/developers/docs/consensus-mechanisms/pow/)
 
-### Proof-of-stake {#proof-of-stake}
+### Casper FFG {#casper-ffg}
 
 Ethereum has plans to upgrade to a **proof-of-stake (PoS)** consensus protocol.
 
 #### Block creation {#pos-block-creation}
 
-Proof-of-stake is done by validators who have staked ETH to participate in the system. A validator is chosen at random to create new blocks, share them with the network and earn rewards. Instead of needing to do intense computational work, you simply need to have staked your ETH in the network. This is what incentivises healthy network behaviour.
+Casper FFG is done by validators who have staked ETH to participate in the system. A validator is chosen at random to create a new block. This randomness constitutes the proposal mechanism that would otherwise be the "work" in proof-of-work for Nakamoto consensus. Once it's a validator's turn to propose a block, they share it with the network and earn rewards. Instead of needing to do intense computational work, they'd simply need to have staked their ETH in the network. This is what incentivises healthy network behaviour as the validator now has skin in the game.
 
 #### Security {#pos-security}
 
-A proof-of-stake system is kept secure by the fact that you'd need 51% of the total staked ETH to defraud the chain. And that your stake is slashed for malicious behaviour.
+A proof-of-stake system is kept secure by the fact that you'd need 34% of the total voting power in a committee to defraud the chain. This encouraged further in that, on Eth2, your stake is slashed for malicious behaviour.
```

Comment:
Wanted to clarify, is this accurate that >1/3 voting power can _defraud_ the chain? Despite being more than half of a supermajority, this still wouldn't give an attacker a supermajority to finalize fraudulent changes, though it could certainly enable a DOS attack by preventing supermajority consensus. 

Also, the "eth2" term is being phased out. Updates to the rest of the site are coming, but I would remove "Eth2" here in favor of "proof-of-stake"

Perhaps:
```suggestion
A proof-of-stake system is kept secure by the fact that you'd need 34% of the total voting power in a committee to defraud the chain. Malicious behavior under proof-of-stake also result in staked ETH being slashed in proportion to the size of the attack.
```